nrt/data/IEEE14/IEEE14_WithAutomata/description/IEEE14_WithAutomata.tex

```diff
@@ -106,7 +109,249 @@ \section{Results}
 \caption{Behavior of the under voltage automaton}
 \end{figure}
 
-This corresponds to the following timeline:
-\lstinputlisting[breaklines, frame = lines, framesep = 2em]{../IEEE14_UnderVoltageAutomaton/reference/outputs/timeLine/timeline.log}
+
+\newpage
+\section{Phase Shifter I}
+\label{PhaseShifterIAutomaton}
+
+The phase-shifter I is an automaton that gradually changes the tap of a phase-shifter transformer if its monitored transiting current goes higher than a defined threshold for a certain amount of time.
+
+\subsection{Initial Conditions}
+
+Initial conditions are the same that in the IEEE14 basic test cases.
+
+\subsection{Models}
+
+Instead of a ratio tap changer transformer (no phase shift) between buses 5 and 6, we use a phase tap changer transformer controlled by a phase-shifter I automaton in order to regulate the transiting current.
+The automaton is supposed to keep the transit within acceptable bounds, by reducing or increasing the phase shift step by step. \\
```

Comment:
I would rather write: to keep the transit under a given threshold, in order to emphasize the difference between the PhaseShifterI (that works with iMax) and PhaseShifterP (that works with a range around PTarget)